libs/sdk-backend-tiger/src/utils/errorHandling.ts

```diff
@@ -58,3 +65,17 @@ function createLimitReachedError(error: Error): LimitReached | undefined {
 
     return new LimitReached("The limit reached. Upgrade your plan to create more objects.", error);
 }
+
+function createContractExpiredError(error: Error): ContractExpired | undefined {
+    const axiosErrorResponse = (error as AxiosError).response;
+
+    if (
+        !axiosErrorResponse ||
+        axiosErrorResponse.status !== 403 ||
+        !axiosErrorResponse.data?.detail?.includes("Contract expired")
```

Comment:
Is this condition safe? Should/does backend rather send some header as a sign of the fact that the contract expired (instead of some text, that may change over time) ?

Reply:
Thanks. We also have the same concern; however, it's unlikely that BE will not provide us with anything other than the text "Contract expired", I think we can only rely on the text to know if an error is thrown because of expired organization.

＊ A sample response when calling an API on an org when it has already expired is as below:

![image](https://user-images.githubusercontent.com/67769750/187851713-64ec4140-90d6-4a95-ae36-1f5301d2410c.png)